app/src/main/res/layout/content_main.xml

```diff
@@ -12,10 +12,9 @@
 
```

Comment:
Could you look at line 10 and 11 in this file 😉 I think you can just remove them...